packages/in_app_purchase/in_app_purchase_android/pubspec.yaml

```diff
@@ -22,10 +22,10 @@ dependencies:
   in_app_purchase_platform_interface: ^1.1.0
   json_annotation: ^4.0.1
   meta: ^1.3.0
-  test: ^1.16.0
 
 dev_dependencies:
   build_runner: ^1.11.1
   flutter_test:
     sdk: flutter
+  test: ^1.16.0
```

Comment:
Please put these in alphabetical order (same for the other files).

Reply:
done